### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by VLeads**
Add homepage

Add header
 
Add footer

rapidquizz

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    width: 70%;
+    margin: auto;
+    background-color: var(--xlight-blue);
+}
+
+a{
+    text-decoration: none;
+}
+
+.page-layout{
```
Comment (abhisheksahani-nova) [inline index.css:18]: page-layout class is empty , yu can remove if  its empty

**Review by abhisheksahani-nova — APPROVED** [review]
else everthing is looking good

File: index.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.footer > a > i{
+    font-size: 3rem;
+}
+
+.footer-header{
+    font-weight: 600
+}
+
+.footer > a > i:hover{
+    color: #383838
```
Comment (deekshithmd) [inline index.css:75]: using variable will be good

**Review by deekshithmd — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (VLeads) [reply index.css:18]: Alright, I'll remove it.

File: index.css (hunk 2 above)
Comment (VLeads) [reply index.css:75]: I've used this because it is only used in one place.

**Pull request merged**